app/views/vm_common/_live_migrate.html.haml

```diff
@@ -52,6 +52,23 @@
                   'paging_div_buttons_state_enabled' => true,
                   'paging_div_buttons_id'            => "angular_paging_div_buttons",
                   'paging_div_buttons_type'          => "Submit"}
+- unless @explorer
+  %table{:width => '100%'}
+    %tr
+      %td{:align => 'right'}
+        #buttons_on
```

Comment:
It looks like none of the others need it. Attach/detach are handled by Angular, and Live Migrate and Evacuate have submittable defaults and are supposed to always be turned on-- there's no checking `@changed`.